core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -294,6 +301,67 @@ public Response mount(@QueryParam("path") String path, @QueryParam("ufsPath") St
     }
   }
 
+  private class MountPoint {
```

Comment:
This type is just to be encoded into json and used by the getMountPoints REST API which is defined just below. Also, this class depends on MountOptions which is defined in the core-server module, if put into core-common/wire module, then this is a dependency loop...